### DISCUSSION TITLE
Adam 1.4 prerelease

### DISCUSSION THREAD

**Discussion by eduardofilo** [eduardofilo/RG350_adam_image#177] [Announcements]
Hi.

We are in test phase of the next Adam revision. Here it can be downloaded a prerelease version in case someone wants to test it:

https://drive.google.com/file/d/1Rbd__zyQjRYWMWGFjNFyD3BWicfrC1lO/view?usp=sharing

Any comment or bug report will be welcome.

This is the change log until now:

* ODbeta updated to 2021-12-12 (surely it solves issue #22).
* RetroArch updated to 2022-01-29.
    * New cores:
        * a5200_libretro.so: Atari 5200. Replaces atari800_libretro in OPK wrapper.
        * vitaquake2_libretro.so (plus three more for mission packs): Quake2
        * jaxe_libretro.so: CHIP-8
        * bk_libretro.so: Elektronika - BK-0010/BK-0011
        * freechaf_libretro.so: Fairchild ChannelF
        * minivmac_libretro.so: Apple Macintosh (no OPK wrapper is included for this one because it lacks core_info and seems immature).
    * Configuration changes:
        * Left stick as D-pad in all RA cores (issue #67).
        * Changed "WHDLoad Support" RA option in Amiga (discussion #136).
* SimpleMenu updated to 10.3. Some configuration changes:
    * New theme [BetaMax](https://github.com/eduardofilo/BetaMax-SimpleMenu-theme) thanks to @Yarkiebrown.
    * Fixed some extensions (.bin for Intellivision and .g64 for Commodore 64) (issue #66).
    * New system Super Game Boy (discussion #122).
    * New system Quake2 based in 4 new RA cores for base game and three mission packs.
    * Fixed SimpleMenu overclocking configuration (issue #175).
    * Changed configuration of Quake to fit [supported games](https://docs.libretro.com/library/tyrquake/#loading-quake-and-expansion-paks) (mission packs).
* FBA updated to [this version](https://github.com/plrguez/fba-sdl/releases/tag/r19_2021-10-18-.44_od_beta-fix-06) thanks to @plrguez.
* PCSX4All updated to [last version](https://github.com/jdgleaver/RG350_pcsx4all/releases/tag/1.5) thanks to @jdgleaver (issue #175).
* Installed uae4all Amiga standalone emulator.
* Replaced ScummVM standalone emulator by [upstreamed version](https://downloads.scummvm.org/frs/scummvm/).
* Installed [this music player](https://github.com/plrguez/gmu/releases/tag/v0.10.1-od) thanks to @plrguez.

**Comment by Xaumir** [comment]
Hey, nice one!

From what I could test, the brightness key combo doesn't work anymore and every time I try to use the overclock funcion my RG350P reboots. I'll keep testing it during the day and will report if I find anything else. Thanks for the hard work!

**Reply by eduardofilo** [reply]
Hi. It's weird. In the RG280V the brightness combo works. I will try when at home in the RG350M.
Overclocking works on few machines.
Thanks for testing it.

**Reply by neilswann80** [reply]
Everything's working well for me so far (RG350m) including the brightness combo.  The new EMU music player is nice to have, I presume we don't have an odbeta compatible video player available yet?

A tip for anyone who wants to name the Doom/Quake/Quake2 games in Simplemenu... just create a text file in each subfolder containing "exec=PAK0.PAK" or "exec=DOOM.WAD" (without the quotes), then rename the .txt file to .fbl.  You can then name the fbl file whatever you want the game to display as and add a png image with this name in .previews. Then just modify the /.simplemenu/section_groups ini file so that "romExts = .fbl"

**Reply by eduardofilo** [reply]
Great tip. I wil use. I At the moment I only used the previews to distinguish them. But this is much better. Thanks.

I haven't found a video player for ODb either.

Thanks for testing the image.

**Reply by eduardofilo** [reply]
@Xaumir I have tested in RG350M and the brightness shortcut also works, like @neilswann80 said. It's not really the same screen, but at least it shows that the key combination works. I'm surprised that it doesn't work on the 350P screen.
Does the brightness level change when you set the value in the GMenu2X options?

**Reply by Xaumir** [reply]
Yeah, it actually does. Before the shortcut was PWR + VOL+ or VOL- right? Now it just controls the volume, and if I continue pressing the power button for a few seconds (even while pressing vol+ or -) it shuts down.

**Reply by neilswann80** [reply]
> Yeah, it actually does. Before the shortcut was PWR + VOL+ or VOL- right? Now it just controls the volume, and if I continue pressing the power button for a few seconds (even while pressing vol+ or -) it shuts down.

@Xaumir the brightness key combo for betaFW should be "Power + left or right D-pad"

**Reply by Xaumir** [reply]
That explains a lot, I'm a dumbass. Thanks! It's working fine.

**Reply by eduardofilo** [reply]
@neilswann80 The tip you gave for Doom/Quake works like a charm. It will come in handy on systems with multiple files of which only one needs to be launched as DOSBox. Thank you very much.

**Reply by neilswann80** [reply]
You're very welcome.  And to return some gratitude, thanks for all the work you do on this image.  It's made using, managing and updating my handheld a pleasure rather than a chore.  It really is like having a new device again.

**Comment by eduardofilo** [comment]
One thing that I noticed is the DOSBox Pure RetroArch core is much more less compatible with DOS games that the one installed in Adam 1.3 (from october 2021). So I will patch that core in final release.
